tools/run_tests/performance/README.md

```diff
@@ -211,27 +214,161 @@ The script [loadtest_config.py](./loadtest_config.py) generates LoadTest
 configurations for tests running a set of scenarios. The configurations are
```

Comment:
I am a little confused about 'Each configurations for tests running a set of scenarios', does that mean we have multiple scenarios in the same tests?

Reply:
I have added a couple more comments making that clear (there was one in the previous paragraph already but I added one at the end of this one).